tests/samples/expected/ocaml.ml

```diff
@@ -931,7 +931,7 @@ let x =
     | true -> 1.
   ]
 
-(* New line for structures in module definitions if they appear in the input *)
+(* New line for structures in module definitions *)
```

Comment:
Does this diff imply that if I write a one-line module a newline will be inserted and it'll become multiline?

Reply:
It's still a softline, so a one-line module will remain on a single line.

Reply:
The diff means that a newline will be inserted if the context is multi-line, even if the newline isn't _precisely there_ in the input (basically it's `scoped_softline` versus `input_softline`). Your one-line module would stay as such.